api/src/main/java/io/opencensus/internal/NullnessUtils.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.internal;
+
+import io.opencensus.common.Function;
+import javax.annotation.Nullable;
+
+/*>>>
+import org.checkerframework.checker.nullness.qual.KeyForBottom;
+*/
+
+/**
+ * Utility methods for suppressing nullness warnings and working around Checker Framework issues.
+ */
+public final class NullnessUtils {
```

Comment:
suggestion: NullnessUtils is a little ambiguous since it not only suppress nullness warning. Consider naming it something like CheckerUtils.

Reply:
I renamed it to CheckerFrameworkUtils.

Reply:
This util somehow reduces code fluentness. Maybe consider using `import static` and shorten the name `removeSuperFromFunctionParameterType`?